Nodejs/Product/Nodejs/Workspace/ContextMenuProvider.cs

```diff
@@ -166,6 +168,11 @@ private void ExecNpmInstallNew(WorkspaceVisualNodeBase node)
                 var workspace = node.Workspace;
                 var packageJson = PackageJsonFactory.Create(((IFileNode)node).FullPath);
 
+                if (string.IsNullOrEmpty(packageJson.Main))
+                {
+                    return;
+                }
```

Comment:
Does this mean the user's request is silently ignored?

Reply:
Yes, there was no good way to show a dialog. I'm talking to AnyCode to see how we can hide the menu item